src/lodash.d.ts

```diff
@@ -0,0 +1 @@
+declare module "lodash";
```

Comment:
I'd rather install `npm i -D @types/lodash` instead of declaring it as a module here.

Reply:
Done